pubspec.yaml

```diff
@@ -4,7 +4,7 @@ publish_to: "none"
 
 environment:
   sdk: ">=3.0.0 <4.0.0"
-  flutter: ">=3.10.0"
+  flutter: ">=3.13.0"
```

Comment:
I tihnk we should fix this to the same version in .tool-versions.

WDYT?

Reply:
Fixing this to 3.13.7 then